CHANGELOG.md

```diff
@@ -1,5 +1,136 @@
+* Retry on connection closed ([#5515](https://github.com/confluentinc/ksql/pull/5515)) ([8eb1f88](https://github.com/confluentinc/ksql/commit/8eb1f88c165aa0bb494288c46cd96ac911d7cfdf))
+* set retention.ms to -1 instead of Long.MAX_VALUE ([#5560](https://github.com/confluentinc/ksql/pull/5560)) ([22da8a0](https://github.com/confluentinc/ksql/commit/22da8a091f0020bbd541ad5f812054c7744f0785))
+* update Concat UDF to new framework and make variadic ([#5513](https://github.com/confluentinc/ksql/pull/5513)) ([cab6a86](https://github.com/confluentinc/ksql/commit/cab6a8672f97d86da75a3e5b1e70985146129640))
+* zero decimal bug ([#5531](https://github.com/confluentinc/ksql/pull/5531)) ([1b9094b](https://github.com/confluentinc/ksql/commit/1b9094bd5ead8a8441ef8f10a5f86722dd7d3988))
+* **client:** expose query ID from executeQuery() return type ([#5391](https://github.com/confluentinc/ksql/pull/5391)) ([bd3c642](https://github.com/confluentinc/ksql/commit/bd3c64289dc315fedd1f5f4040d571a5bf26323a))
+* **client:** fail queryID future if request fails (MINOR) ([#5435](https://github.com/confluentinc/ksql/pull/5435)) ([ddaf9f9](https://github.com/confluentinc/ksql/commit/ddaf9f998009cf58ee68133dac143191c0e1ad9a))
+* /query-stream endpoint should serialize Struct (MINOR) ([#5205](https://github.com/confluentinc/ksql/pull/5205)) ([12b092b](https://github.com/confluentinc/ksql/commit/12b092bda160f614ef0b8096f77b7013efc3b190))
+* Move Cors handler in front of /chc handlers ([#5239](https://github.com/confluentinc/ksql/pull/5239)) ([004ced2](https://github.com/confluentinc/ksql/commit/004ced255479f6a5975848270746a45746a4437a))
+* use schema in annotation as schema provider if present ([1a90eeb](https://github.com/confluentinc/ksql/commit/1a90eeb71ae8292410fc99d49c24fdafadcf9de7))
+* use sr's jackson-jsonschema version ([#5213](https://github.com/confluentinc/ksql/pull/5213)) ([0b3899a](https://github.com/confluentinc/ksql/commit/0b3899a4a31f668a897d82d98c59365aae8022ec))
+
+
+
+### Reverts
+
+* Revert "build: reference plugin URL as protperty (MINOR) (#5497)" ([093955f](https://github.com/confluentinc/ksql/commit/093955f9ce875a0b8656aaec78d4c619ebdc83ff)), closes [#5497](https://github.com/confluentinc/ksql/issues/5497)
+* Revert "build ksqlDB against 6.0.0 (#5339)" (#5347) ([0a1bdad](https://github.com/confluentinc/ksql/commit/0a1bdadb7745efd2d3a5859ab5018719a2ed7f1e)), closes [#5339](https://github.com/confluentinc/ksql/issues/5339) [#5347](https://github.com/confluentinc/ksql/issues/5347)
+* Any key name (#5093) ([1f0ca3e](https://github.com/confluentinc/ksql/commit/1f0ca3efb0f5ecadc7a604f219cbc16b2e120d8d)), closes [#5093](https://github.com/confluentinc/ksql/issues/5093)
+
+
+
+### BREAKING CHANGES
+
+* Statements containing PARTITION BY, GROUP BY or JOIN clauses will now produce different output schemas.

```

Comment:
Are all the lines below this one from the same breaking change? If so they need to be indented so the markdown formatting is correct.

We should probably update the changelog script to do this automatically. I'll look into that for the next release.